harvesting/util.py

```diff
@@ -67,3 +67,10 @@ def db_connect():
                                connectTimeoutMS=100,
                                serverSelectionTimeoutMS=100
                                )[os.environ.get('DATABASE_NAME')]
+
+def rate_limit_reached(requests_response):
+    try:
+        rate_limit_check = requests_response.headers.get('x-ratelimit-remaining')
+        if int(rate_limit_check) < 10: return True
```

Comment:
As error reported at #434, sometimes the header does not exist and the int conversion fails.

Please add a None check.
```suggestion
        if rate_limit_check is not None and int(rate_limit_check) < 10: return True
```

Reply:
Thanks for the tips! A new issue #470 was created from a branch which supersedes this pull request. The reported error after the fix could not be reproduced so an extra test was added to verify that the fix worked (harvesting/util_unittest.py)